src/main/java/com/javapractice/controllers/TodoController.java

```diff
@@ -0,0 +1,33 @@
+package com.javapractice.controllers;
+
+import org.springframework.web.bind.annotation.RestController;
+
+import com.javapractice.domain.Request.TodoRequest;
+import com.javapractice.domain.Response.TodoResponse;
+import com.javapractice.interactors.interfaces.IaddTodo;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+
+@RestController
+class TodoController {
+	private final IaddTodo addTodoInteractor;
+
+	//constructor that injects addTodoInteractor
+	public TodoController(IaddTodo addTodoInteractor) {
+		this.addTodoInteractor = addTodoInteractor;
```

Comment:
As you can see, the controller has the interactor's injected, similar to how the repository is injected into the interactor.